t/proxyunits.t

```diff
@@ -411,10 +459,44 @@ check_version($ps);
     print $be "TOUCHED\r\n";
 
     is(scalar <$ps>, "TOUCHED\r\n", "got TOUCHED instread of STORED");
+}
 
-    # TODO: meta quiet cases
-    # - q should be turned into a space on the backend
-    # - errors should still pass through to client
+{
+    subtest 'quiet flag 1' => sub {
+        # Test Scenario
+        my $ps_send = "ms /b/a 2 q\r\nhi\r\n";
+        # backend must receive response with quiet flag replaced by a space.
+        my %be_recv = (0 => ["ms /b/a 2  \r\n", "hi\r\n"]);
+        my %be_send = (0 => ["HD\r\n"]);
+        # proxy must not return HD response from the backend.
+        my @ps_recv = ();
+        proxy_test($ps_send, \%be_recv, \%be_send, \@ps_recv);
```

Comment:
Doesn't seem like this asserts that there's no response?

This is where check_version comes in handy (or not trying to wrap everything in one function); assert that you didn't see the HD because you did see a VERSION that was sent later.

Reply:
`proxy_test` could check for empty response if @ps_recv is empty easy enough though; `if (@{$ps_recv}) { do the check } else { check_version }`

Reply:
Options:
1. Rely on external `check_sanity()` at section boarder.
2. Add `check_sanity()` at the end of `proxy_test` to always check both proxy and backends. 
3. Add `check_version()` like you suggested
4. Add `check_version` regardless if ps_recv is empty or not.

I am following your suggestion, but if you see any merit in another option, please let me know.

Reply:
I think if the point of the test is to verify a non-response, it should be encompassed within the pass-fail of the subtest. I'd prefer to not add `check_sanity()` at the end of _every_ subtest since that will slow down the suite eventually, otherwise I don't really care.

Reply:
`check_version` added.